### ISSUE TITLE
Implement Oracle mount tests

### ISSUE THREAD

**Issue by sebastienros** [dotnet/aspire#5188]
Add `WithInitBindMount()` and `WithDataBindMount()` test coverage for Oracle.

Reference implementation:

https://github.com/dotnet/aspire/pull/4943/commits/73b6233cd0c074c778a3458cc45761d92d91ae91#diff-dc94545ae8c6bec88f2588c087937d6ec7db6298cae1946acbeb3b525d381ef3L388

Several issues on the CI made us postpone this work to unblock test coverage.
